packages/istanbul-reports/lib/html/annotator.js

```diff
@@ -41,9 +41,11 @@ function annotateLines(fileCoverage, structuredText) {
 }
 
 function annotateStatements(fileCoverage, structuredText) {
-    var statementStats = fileCoverage.s,
-        statementMeta = fileCoverage.statementMap;
-    Object.keys(statementStats).forEach(function (stName) {
+    var statementStats = Object.keys(fileCoverage.s).filter(function (stName) {
+      return fileCoverage.s[stName] !== null;
+    }),
+    statementMeta = fileCoverage.statementMap;
```

Comment:
I think this should be a `var statementMeta` line, it's kind of hard to grok imho.